packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts

```diff
@@ -812,6 +824,47 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
     }
   }
 
+  const fetchBuyQuote = function* ({ payload }: ReturnType<typeof A.fetchBuyQuote>) {
+    while (true) {
+      try {
+        yield put(A.fetchBuyQuoteLoading())
+
+        const { amount, pair, paymentMethod } = payload
+        const pairReversed = reversePair(pair)
+        let paymentMethodId: string | undefined
+
+        if (paymentMethod === BSPaymentTypes.BANK_TRANSFER) {
+          paymentMethodId = 'd74992e0-a462-4aeb-903e-fc8e9a11bb40'
```

Comment:
is this some defined payment method? might be good to move it to const and add comments what it represents

Reply:
Yes it is, moved to const and added comments

Reply:
I confirmed with backend and it's not predefined payment method. Added the fix, should be dynamic now